src/Catalyst.Modules.Server.Blazor/Components/TransactionTableComponent.razor.cs

```diff
@@ -33,6 +33,6 @@ public class TransactionTableComponentBase : ComponentBase
     {
         [Parameter] public TransactionBroadcastDao Model { get; set; }
 
-        public string GetAmount() { return Model.PublicEntries.First().Amount; }
+        // public string GetAmount() { return Model.PublicEntries.First().Amount; }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4600656)